src/GitLink/Program.cs

```diff
@@ -24,51 +27,51 @@ private static int Main(string[] args)
             var consoleLogListener = new OutputLogListener();
             LogManager.AddListener(consoleLogListener);
 
-            try
+            Uri remoteGitUrl = null;
+            string pdbPath = null;
+            bool skipVerify = false;
+            bool downloadWithPowershell = false;
+            var arguments = ArgumentSyntax.Parse(args, syntax =>
             {
-                HelpWriter.WriteAppHeader(s => Log.Write(LogEvent.Info, s));
+                syntax.DefineOption("u|url", ref remoteGitUrl, s => new Uri(s, UriKind.Absolute), "Url to remote git repository.");
```

Comment:
We need to keep:

- b|branch
- s|shaHash
- errorsaswarnings
- skipverify (so don't remap s to skipverify)
- l|file (for logging purposes)
- debug